src/plugins/plot/inspector/forms/SeriesForm.vue

```diff
@@ -84,10 +88,15 @@
       </li>
       <li class="grid-row">
         <div class="grid-cell label" title="Display markers visually denoting points in alarm.">
-          Alarm Markers
+          <label for="alarm-markers-checkbox">Alarm Markers</label>
```

Comment:
@ozyx here's an attempt at using labels and ids

Reply:
what happens in the case of an overlay plot with multiple series to configure?